tyr-core/src/main/java/org/jboss/tyr/check/SkipCheck.java

```diff
@@ -37,6 +37,9 @@ public boolean shouldSkip(JsonObject payload, FormatYaml config) throws InvalidP
         if (payload == null || config == null) {
             throw new IllegalArgumentException("Input arguments cannot be null");
         }
+        if (gitHubService.commitCheckDisabled()) {
```

Comment:
come on, you can write this conditional into the existing if ;)

Reply:
Implemented fix based on recommendation.